wolfssl/openssl/crypto.h

```diff
@@ -96,13 +96,8 @@ WOLFSSL_API int wolfSSL_OPENSSL_init_crypto(word64 opts, const OPENSSL_INIT_SETT
 #define SSLeay_version wolfSSLeay_version
 #define SSLeay wolfSSLeay
 #define OpenSSL_version_num wolfSSL_OpenSSL_version_num
+#define SSLEAY_VERSION_NUMBER OPENSSL_VERSION_NUMBER
 
-#if defined(WOLFSSL_QT) || defined(WOLFSSL_HITCH)
```

Comment:
We'll need to confirm this does not break QT or Hitch. I agree with this change but, we might need to also adjust openssl.h.

Reply:
@julek-wolfssl or @kareem-wolfssl do either of you know if this will break QT or Hitch OSP projects?

Reply:
I don't see it being used in the latest versions of either lib. This should be safe.

Reply:
Also the hitch CI is passing.